subsystems/provisioning/bluetooth/bluetooth_wifi_provisioner.go

```diff
@@ -0,0 +1,155 @@
+// Package ble contains an interface for using bluetooth-low-energy to retrieve WiFi and robot part credentials for an unprovisioned Agent.
+package ble
+
+import (
+	"context"
+	"encoding/json"
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
[nit] Not sure what agent's pattern is, but generally the standard library `errors` is a little more idiomatic.